### DISCUSSION TITLE
can this be run on pero if so how?

### DISCUSSION THREAD

**Discussion by ThnksCJ** [Chevereto/Chevereto-Free#126] [Help needed]
how would i run this on ptero i would like to add this to my vps

**Comment by rodber** [comment]
What's pero?

**Comment by ThnksCJ** [comment]
Pterodactyl panel

**Reply by rodber** [reply]
> Pterodactyl panel

You should ask them about that, the requirements are trivial anyway. Easy as place the application files in a exposed PHP web server and profit.

https://chevereto-free.github.io/setup/system/requirements.html
